org.opent2t.onboarding.getverahub/js/thingOnboarding.js

```diff
@@ -0,0 +1,120 @@
+'use strict';
+var https = require('https');
+var cryptojs_sha1 = require('crypto-js');
+var request = require('request-promise');
+
+class Onboarding {
+
+    ///
+    /// gets the session token for the provided server url
+    ///
+    getSessionToken(url) {
+        return this.makeVeraRequest(url, this.veraSessionTokenPath, false, false, this.mmsInfo, true);
+    }
+
+    ///
+    /// gets the mms info for the user account
+    ///
+    getMmsAuthInfo() {
+        var path = '/autha/auth/username/' + this.username + '?SHA1Password=' + this.sha1Password + '&PK_Oem=1';
```

Comment:
opent2t isn't consistent on this, we should make a call one way or the other

---
In reply to: [103515998](https://github.com/openT2T/onboarding/pull/32#discussion_r103515998) [](ancestors = 103515998)